cowprotocol/watch-tower
# feat: observability (api, metrics, and startup / liveliness probes)

# Description
This PR adds observability to the watch-tower so that it can be monitored in production.

# Changes

- [x] Add express API with `/api/dump` endpoint
- [x] Add prometheus metrics at `/metrics`
- [ ] Add startup and liveliness probes at `/health`
- [ ] Add CLI options for enabling API (enabled by default)

## How to test

1. Start watch-tower.
2. Check above endpoints. For `dump`, add chain Id, for example: `/api/dump/1`

## Related Issues

Fixes #70
